MobileSyncExplorerReactNative/ios/Podfile

```diff
@@ -7,40 +10,8 @@ target 'MobileSyncExplorerReactNative' do
 
 source 'https://cdn.cocoapods.org/'
 
-pod 'FBLazyVector', :path => '../node_modules/react-native/Libraries/FBLazyVector'
-pod 'FBReactNativeSpec', :path => '../node_modules/react-native/Libraries/FBReactNativeSpec'
-pod 'RCTRequired', :path => '../node_modules/react-native/Libraries/RCTRequired'
-pod 'RCTTypeSafety', :path => '../node_modules/react-native/Libraries/TypeSafety'
-pod 'React', :path => '../node_modules/react-native/'
-pod 'React-Core', :path => '../node_modules/react-native/'
-pod 'React-CoreModules', :path => '../node_modules/react-native/React/CoreModules'
-pod 'React-Core/DevSupport', :path => '../node_modules/react-native/'
-pod 'React-RCTActionSheet', :path => '../node_modules/react-native/Libraries/ActionSheetIOS'
-pod 'React-RCTAnimation', :path => '../node_modules/react-native/Libraries/NativeAnimation'
-pod 'React-RCTBlob', :path => '../node_modules/react-native/Libraries/Blob'
-pod 'React-RCTImage', :path => '../node_modules/react-native/Libraries/Image'
-pod 'React-RCTLinking', :path => '../node_modules/react-native/Libraries/LinkingIOS'
-pod 'React-RCTNetwork', :path => '../node_modules/react-native/Libraries/Network'
-pod 'React-RCTSettings', :path => '../node_modules/react-native/Libraries/Settings'
-pod 'React-RCTText', :path => '../node_modules/react-native/Libraries/Text'
-pod 'React-RCTVibration', :path => '../node_modules/react-native/Libraries/Vibration'
-pod 'React-Core/RCTWebSocket', :path => '../node_modules/react-native/'
-
-pod 'React-cxxreact', :path => '../node_modules/react-native/ReactCommon/cxxreact'
-pod 'React-jsi', :path => '../node_modules/react-native/ReactCommon/jsi'
-pod 'React-jsiexecutor', :path => '../node_modules/react-native/ReactCommon/jsiexecutor'
-pod 'React-jsinspector', :path => '../node_modules/react-native/ReactCommon/jsinspector'
-pod 'React-callinvoker', :path => '../node_modules/react-native/ReactCommon/callinvoker'
-pod 'ReactCommon/turbomodule/core', :path => '../node_modules/react-native/ReactCommon'
-pod 'Yoga', :path => '../node_modules/react-native/ReactCommon/yoga', :modular_headers => true
-
-pod 'DoubleConversion', :podspec => '../node_modules/react-native/third-party-podspecs/DoubleConversion.podspec'
-pod 'glog', :podspec => '../node_modules/react-native/third-party-podspecs/glog.podspec'
-pod 'Folly', :podspec => '../node_modules/react-native/third-party-podspecs/Folly.podspec'
-
-pod 'RNVectorIcons', :path => '../node_modules/react-native-vector-icons'  
-pod 'RNGestureHandler', :path => '../node_modules/react-native-gesture-handler'
-pod 'react-native-safe-area-context', :path => '../node_modules/react-native-safe-area-context'
+config = use_native_modules!
```

Comment:
This function (which comes from https://github.com/react-native-community/cli/blob/master/packages/platform-ios/native_modules.rb) iterates through the dependencies and automatically add the pods (the so-called "auto-linking"). I'll take a look, maybe it's trivial to make it work.